plugin/core/protocol.py

```diff
@@ -6121,6 +6123,13 @@ def to_lsp(self) -> 'Position':
         }
 
 
+ResponseError = TypedDict('ResponseError', {
+    'code': int,
+    'message': str,
+    'data': NotRequired['LSPAny']
+})
```

Comment:
It looks like it is not declared in https://github.com/sublimelsp/lsp-python-types/blob/main/lsprotocol/lsp.json

And because of that, the generator script doesn't generate the type

Reply:
Yeah, I wonder whether this is intentionally not declared in the defintions, or was just missed. But maybe the reason is that this is more of a general type from JSON-RPC, and not directly from the language server protocol.